build.gradle

```diff
@@ -236,6 +237,8 @@ dependencies {
 
     compile group: 'com.google.guava', name: 'guava', version: '27.0-jre'
 
+    compile group: 'org.meanbean', name: 'meanbean', version: '2.0.3'
+    compile group: 'joda-time', name: 'joda-time', version: '2.9.4'
```

Comment:
We should use the standard Java 8 JSR 310 Time API rather than adding a new library
https://www.oracle.com/technetwork/articles/java/jf14-date-time-2125367.html

That said, it doesn't look like anything in this PR imports anything from either of these two libraries. Can they be removed?